lib/upload.js

```diff
@@ -974,11 +995,10 @@ function resolveUrl (origin, link) {
  *
  * @param {number} totalSize The byte size of the upload, which will be split.
  * @param {number} partCount The number in how many parts the upload will be split.
- * @param {string[]} previousUrls The upload URLs for previous parts.
  * @return {object[]}
  * @api private
  */
-function splitSizeIntoParts (totalSize, partCount, previousUrls) {
+function splitSizeIntoParts (totalSize, partCount) {
```

Comment:
Didn't see a need for `previousUrls` to be a parameter here, so I factored it out for better separation of concerns. I can revert this change since my PR is not using a callback anymore based off this function. I think the change is an improvement, though.